src/main/java/slimeknights/tconstruct/library/materials/stats/BaseMaterialStats.java

```diff
@@ -14,40 +15,25 @@
 public abstract class BaseMaterialStats implements IMaterialStats {
 
   @Override
-  public String getLocalizedName() {
-    // todo
-    return null;
-  }
-
-  @Override
-  public List<String> getLocalizedInfo() {
-    // todo
-    return null;
-  }
-
-  @Override
-  public List<String> getLocalizedDesc() {
-    // todo
-    return null;
+  public ITextComponent getLocalizedName() {
+    return new TranslationTextComponent(String.format("stat.%s.name", this.getIdentifier().getPath()));
   }
 
-  public static String formatNumber(String loc, String color, int number) {
+  public static ITextComponent formatNumber(String loc, String color, int number) {
     return formatNumber(loc, color, (float) number);
   }
 
-  public static String formatNumber(String loc, String color, float number) {
-    return String.format("%s: %s%s%s",
-      Util.translate(loc),
-      color,
-      Util.df.format(number),
-      TextFormatting.RESET);
+  public static ITextComponent formatNumber(String loc, String color, float number) {
+    return new TranslationTextComponent(loc)
+      .appendSibling(new StringTextComponent(": "))
```

Comment:
Might as well use append string here. My comment on Discord was if we do it using appendSibling, we can then format the text component we pass into appendSibling:

```java
return new TranslationTextComponent(loc)
    .appendSibling(new StringTextComponent("whatever").applyTextStyle(TextFormatting.WHITE));
```

Since we could not get color working in 1.15, might as well stick to `appendString` for now.